xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/layouts/livedata-table.html

```diff
@@ -0,0 +1,90 @@
+<template>
+    <table class="xwiki-livetable">
+
+        <thead>
+
+            <tr>
+
+                <th
+                    v-for="(col, cIdx) in cols"
+                    :key="cIdx"
+                >
+                    <div class="column-header">
+                        <div>{{ col.name }}</div>
```

Comment:
Do you need the 2 DIVs? Why not put the ``column-header`` class on the TH? Why do you need an additional wrapping for the column name?

Reply:
OK, wrapping the column name makes sense when the table header can contain something else, but that means one DIV. I would still put the ``column-header`` class on the TH if it's possible and if you don't have a good reason to keep it.